apps/costcontrol/test/unit/fte_test.js

```diff
@@ -59,8 +59,9 @@ suite('FTE Test Suite >', function() {
   });
 
   teardown(function() {
+    ConfigManager.supportCustomizeMode = false;
```

Comment:
Maybe it's better to add this to the setup of the test and override when needed.